app/models/abilities/administrator.rb

```diff
@@ -82,6 +82,8 @@ def initialize(user)
 
       can [:create], Document
       can [:create, :destroy], DirectUpload
+
+      can [:deliver], Newsletter, hidden_at: nil
```

Comment:
Not sure we should specify all the other actions an admin can do over a newsletter 🤔  (like create, edit, update..) Maybe I'm not aware of how default cancancan works